charts/kong/README.md

```diff
@@ -686,6 +686,8 @@ Kong is going to be deployed.
 
 #### Kong Enterprise Docker registry access
 
+:bulb: This is needed only for Kong Enterprise version <=2.2 - since 2.3 the image can be downloaded without those pullSecrets.
```

Comment:
```suggestion
:bulb: This is needed only for Kong Enterprise versions older than 2.3 - from 2.3 onwards, the image can be downloaded without those pullSecrets.
```

This is because, technically, "2.2.0.1" is not "<= 2.2". My understanding is that for any 2.2.x.y these steps must be followed - please correct me if I'm wrong.